metis-enrichment/metis-enrichment-client/src/test/java/eu/europeana/enrichment/utils/DereferenceUtilsTest.java

```diff
@@ -54,648 +44,598 @@
-        
-        proxy.setChoiceList(choiceList);
-        
-        ArrayList<ProxyType> proxyList = new ArrayList<ProxyType>();
-        proxyList.add(proxy);
-        
-        rdf.setProxyList(proxyList);
-        
-        Set<String> result = DereferenceUtils.extractValuesForDereferencing(rdf);
-		
-		Assert.assertNotNull(result);
-        Assert.assertEquals(35, result.size());
-        
-        Assert.assertTrue(result.contains("http://dummy1.dum"));
-        Assert.assertTrue(result.contains("http://dummy2.dum"));
-        Assert.assertTrue(result.contains("http://dummy3.dum"));
-        Assert.assertTrue(result.contains("http://dummy4.dum"));
-        Assert.assertTrue(result.contains("http://dummy5.dum"));
-        Assert.assertTrue(result.contains("http://dummy6.dum"));
-
-        Assert.assertTrue(result.contains("http://dummy7.dum"));
-        Assert.assertTrue(result.contains("http://dummy8.dum"));
-        Assert.assertTrue(result.contains("http://dummy9.dum"));
-        Assert.assertTrue(result.contains("http://dummy10.dum"));
-        Assert.assertTrue(result.contains("http://dummy11.dum"));
-        Assert.assertTrue(result.contains("http://dummy12.dum"));
-        
-        Assert.assertTrue(result.contains("http://dummy13.dum"));
-        Assert.assertTrue(result.contains("http://dummy14.dum"));
-        Assert.assertTrue(result.contains("http://dummy15.dum"));
-        Assert.assertTrue(result.contains("http://dummy16.dum"));
-        Assert.assertTrue(result.contains("http://dummy17.dum"));
-        Assert.assertTrue(result.contains("http://dummy18.dum"));
-        
-        Assert.assertTrue(result.contains("http://dummy19.dum"));
-        Assert.assertTrue(result.contains("http://dummy20.dum"));
-        Assert.assertTrue(result.contains("http://dummy21.dum"));
-        Assert.assertTrue(result.contains("http://dummy22.dum"));
-        Assert.assertTrue(result.contains("http://dummy23.dum"));
-        Assert.assertTrue(result.contains("http://dummy24.dum")); 
-
-        Assert.assertTrue(result.contains("http://dummy25.dum"));
-        Assert.assertTrue(result.contains("http://dummy26.dum"));
-        Assert.assertTrue(result.contains("http://dummy27.dum"));
-        Assert.assertTrue(result.contains("http://dummy28.dum"));
-        Assert.assertTrue(result.contains("http://dummy29.dum"));
-        Assert.assertTrue(result.contains("http://dummy30.dum"));
-        
-        Assert.assertTrue(result.contains("http://dummy31.dum"));
-        Assert.assertTrue(result.contains("http://dummy32.dum"));
-        Assert.assertTrue(result.contains("http://dummy33.dum"));
-        Assert.assertTrue(result.contains("http://dummy34.dum"));
-        Assert.assertTrue(result.contains("http://dummy35.dum"));
-    }
+
+  @Test
+  public void testPlaceListExtractedValues() {
+    RDF rdf = new RDF();
+
+    ProxyType proxy = new ProxyType();
+    ArrayList<ProxyType> proxyList = new ArrayList<ProxyType>();
```

Comment:
Don't need explicit type argument in the right hand side of this declaration. This happens multiple times in this class.